src/data-display/board-item/PBoardItem.vue

```diff
@@ -128,6 +128,7 @@ export default defineComponent<BoardItemProps>({
         } = useSelect({
             selected: computed(() => props.selected),
             value: computed(() => props.value),
+            predicate: () => Boolean(props.selected && props.value && props.selected === props.value),
```

Comment:
nitpick: what about this~?

```suggestion
            predicate: () => !!(props.selected && props.value && props.selected === props.value)
```